packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs

```diff
@@ -155,20 +155,30 @@
     
   </Fragment>
   
-    <Fragment>
-    <DirectoryRef Id="ApplicationProgramsFolder">
+  <Fragment>
+    <ComponentGroup Id="ApplicationShortcuts" Directory="ApplicationProgramsFolder">
       <Component Id="ApplicationShortcut" Guid="E05F8D29-F530-4791-AE6B-B89DAE1F6D5C">
-        <Shortcut Id="ApplicationStartMenuShortcut"
-             Name="Keybase CMD"
-           Description="Open command console with keybase in the local path"
-            Target="[SystemFolder]cmd.exe"
-            Arguments="/K &quot;set PATH=%PATH%;[INSTALLFOLDER]&quot;"
-            WorkingDirectory="INSTALLFOLDER"      
-            Icon="ProductIcon"/>
-        <RemoveFolder Id="ApplicationProgramsFolder" On="uninstall"/>
-        <RegistryValue Root="HKCU" Key="Software\Keybase\Keybase" Name="installed" Type="integer" Value="1" KeyPath="yes"/>
+          <Shortcut Id="ApplicationStartMenuShortcut"
+               Name="Keybase CMD"
```

Comment:
this is what github does for its ui and i think the naming is better. it is a shell with keybase access